allennlp/nn/activations.py

```diff
@@ -99,5 +99,4 @@ def _get_name(self):
     "softsign": (torch.nn.Softsign, None),
     "tanhshrink": (torch.nn.Tanhshrink, None),
     "selu": (torch.nn.SELU, None),
-    "gelu": (torch.nn.GELU, None),
```

Comment:
Why remove this?

Reply:
It's a duplicate. It's already there a few lines up.

Reply:
Ah, good catch